contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -41,12 +40,10 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
     address public constant BURNED_TX_FEES_ADDRESS = 0x0100000000000000000000000000000000000000;
 
     /**
-     * @notice Designated Blackhole Address for this contract.
-     *
-     * @dev Tokens are sent here to be "burned" before sending an unlock message to the source chain. Different
-     * from the burned tx fee address so they can be tracked separately.
+     * @dev Used when reporting burned gas fees to the source chain. Fees will be unlocked and sent to this address.
+     * Different from the burned tx fee address so they can be tracked separately.
```

Comment:
`Used when reporting burned gas fees` seems more appropriate for `BURNED_TX_FEES_ADDRESS`

Are fees handled by this address? My understanding is that it's used to
1) burn tokens on the destination to be unlocked on the source
2) burn tokens on the source when reporting burned fees